articles/stay-on-course-with-your-security-compliance-goals.md

```diff
@@ -0,0 +1,54 @@
+
+## Set your goals
+
+One of the reasons compliance is so complicated is that there isn’t a single set of rules. You have to determine what compliance means for your organization.
+
+Are you creating internal processes and controls to stay secure? Is there a law or regulation required to do business? Which industry standards should you meet if you truly want to compete?
+
+Answering questions like these is the first step to becoming compliant. After all, you can’t measure progress until you know where you’re going.
+
+## Be realistic
+
+Achieving the highest level of compliance for your industry is an admirable goal. But pursuing compliance perfection could tie up daily business operations.
+
+For instance, if your organization has 30 employees who use MacBooks, it might be possible to make sure they all have the latest version of macOS installed. That goal gets more challenging to achieve as those MacBooks multiply to the hundreds or even thousands. Maybe then you just want to make sure they’re on Big Sur.
+
+Be sure to set goals based on the resources you have available, and try not to let perfection stand in the way of progress.
+
+## Limit the scope
+
+While your industry may have compliance standards, these recommendations may not apply to your entire organization.
+
+Let’s look at the Payment Card Industry Data Security Standard or PCI DSS. This is a set of standards that aims to protect debit and credit card transactions against data theft and fraud. This is pretty important if your organization plans to accept card payments. But how many of your teams are in the position to collect payments — or even interact with customers?
+
+There may be processes that apply to your entire company, like using multi-factor authentication. But this isn’t always the case. Knowing which compliance standards apply to which teams will help lighten the load across your organization.
+
+## Delegate to team leads
+
+Ultimately, your Head of Security or Chief Compliance Officer is responsible for ensuring compliance, depending on the structure of your organization. Asking them to check the progress of each team and individual, in addition to their other responsibilities, isn’t exactly efficient (or fair).
+
+Your company could hire more security specialists. Some organizations have thousands on staff, but we know this isn’t always a realistic option. The good news is that you don’t need more security people — just more people with security skills.
+
+Team leads are in the perfect position to help you measure compliance progress. They know which hardware needs maintenance. They know which software to update, and they’ll be the first to know of any employees who are onboarding or offboarding.
```

Comment:
I would replace this with something about how they know the technology. For example "Your lead devops engineers know exactly how your cloud environments are configured at scale, and when given visibility into what needs to be done, are the people with the best skills to get it done properly. Provide your technical experts with the reasoning behind the compliance requirements as well as with the visibility they need to know which systems are or aren't compliant."

Reply:
Revised to focus on partnering with technical experts.